src/mocks/prisma.mock.ts

```diff
@@ -0,0 +1,46 @@
+export const createPrismaMock = () => {
+  const connectMock = jest.fn();
+  const disconnectMock = jest.fn();
+
+  const createMock = jest.fn();
+  const deleteMock = jest.fn();
+  const deleteManyMock = jest.fn();
+  const findManyMock = jest.fn();
+  const findOneMock = jest.fn();
+  const updateMock = jest.fn();
+
+  const prisma = {
+    $connect: connectMock,
+    $disconnect: disconnectMock,
```

Comment:
The latest version of Prisma added the `$` to denote default methods on the client to prevent overlapping with generated models. This mock simply implements that.

Reply:
Ah - got it. Thanks for helping me understand.